packages/gas-report/ts/index.ts

```diff
@@ -130,7 +130,7 @@ async function runGasReport(options: Options): Promise<GasReport> {
   const lines = logs.split("\n").map(stripAnsi);
   const gasReportPattern = /^\s*GAS REPORT: (\d+) (.*)$/;
   const testFunctionPattern = /^\[(?:PASS|FAIL).*\] (\w+)\(\)/;
-  const testFilePattern = /^Running \d+ tests? for (.*):(.*)$/;
+  const testFilePattern = /^Ran \d+ tests? for (.*):(.*)$/;
```

Comment:
nice catch! thoughts on testing for both instead of just one? so we can support older foundry versions for a little while? maybe with a comment explaining

Reply:
Sure! I have amended the commit.